test/processStates_test.js

```diff
@@ -0,0 +1,261 @@
+goog.module('datalayerhelper.helper.testing.processStates');
+goog.setTestOnly();
+
+const {DataLayerHelper, buildAbstractModelInterface_} = goog.require('helper');
+
+describe('The processStates function', function() {
```

Comment:
We probably should stick to arrow functions (especially that it's been used below)